docs/collector.aspnet_aspnetapplications.md

```diff
@@ -0,0 +1,29 @@
+# aspnet_aspnetapplications collector
+
+The aspnet_aspnetapplications collector exposes metrics about ASP.NET Applications.
+
+|||
+-|-
+Metric name prefix  | `aspnet_aspnetapplications`
+Classes             | `Win32_PerfRawData_ASPNET_ASPNETApplications"`
+Enabled by default? | No
+
+## Flags
+
+None
+
+## Metrics
+
+Name | Description | Type | Labels
+-----|-------------|------|-------
+`wmi_...` | ... | counter/gauge/histogram/summary | ...
+`wmi_aspnet_aspnetapplications_percent_managed_processor_timeestimated ` | This allows you to get the estimated CPU time for your specific application, and not the entire IIS application pool which could be multiple applications. | gauge | `process`
```

Comment:
Please fill in metrics exposed. It's okay to not provide a long description for all of them, but they should be listed at least.